app/geant-exporter/PrimaryGeneratorAction.hh

```diff
@@ -8,21 +8,20 @@
 
 #pragma once
 
-// Geant4
-#include "G4VUserPrimaryGeneratorAction.hh"
-#include "G4Event.hh"
-#include "G4ParticleGun.hh"
+#include <memory>
+
+#include <G4VUserPrimaryGeneratorAction.hh>
+#include <G4Event.hh>
+#include <G4ParticleGun.hh>
 
 class PrimaryGeneratorAction : public G4VUserPrimaryGeneratorAction
 {
   public:
     PrimaryGeneratorAction();
-    virtual ~PrimaryGeneratorAction();
-
-    virtual void GeneratePrimaries(G4Event* event);
+    ~PrimaryGeneratorAction() override;
```

Comment:
I'm surprised the compiler didn't complain about this one: destructors/constructors don't exactly override the parent. Since the destructor is "default" anyway, I think you can just delete this destructor declaration and the definition that corresponds in the .cc file.

By the way, can you add warning flags to your build script? Take a look at the latest `yuri.cmake`:
```cmake
# Options
set(CELERITAS_USE_CUDA OFF CACHE BOOL "")
set(CELERITAS_USE_Geant4 ON CACHE BOOL "")
set(CELERITAS_USE_ROOT ON CACHE BOOL "")
set(CELERITAS_USE_VECGEOM ON CACHE BOOL "")

# Add all the warnings, and enable color diagnostics when using Ninja
set(CMAKE_CXX_FLAGS "-Wall -Wextra -pedantic -Werror -fcolor-diagnostics" CACHE STRING "")
```